packages/helper/src/socket/index.ts

```diff
@@ -30,8 +30,8 @@ const connectSocket = (app): any => {
 		},
 	});
 
-	server.listen(8081, () => {
-		logger.info(`socket listening on: ${8081}`);
+	server.listen(3010, () => {
```

Comment:
probably best to put this in an env

Reply:
probably, we might move socket stuff into the service directly tho since it's actually not used anywhere else yet